src/main/rx/crsf.c

```diff
@@ -56,6 +56,8 @@
 
 #define CRSF_PAYLOAD_OFFSET offsetof(crsfFrameDef_t, type)
 
+#define CRSF_LINK_STATUS_UPDATE_US                   250000 // 250ms, 4 Hz mode 1 telemetry
```

Comment:
I'd rename that to `CRSF_LINK_STATUS_UPDATE_TIMEOUT_US` or similar to indicate that this is a timeout value.

Reply:
done